### ISSUE TITLE
[BUG] Bugs and Features

### ISSUE THREAD

**Issue by itsdinkd** [hypherionmc/sdlink#4]
latest version on 1.18.1

```java
[18:06:28] [Server thread/WARN]: handleDisconnection() called twice
say test
[18:16:53] [Server thread/ERROR]: Exception caught during firing event: begin 4, end 3, length 3
        Index: 3
        Listeners:
                0: HIGH
                1: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/CommandEvent;)V
                2: NORMAL
                3: ASM: me.hypherionmc.sdlink.events.ServerEvents@3a0dd465 commandEvent(Lnet/minecraftforge/event/CommandEvent;)V
java.lang.StringIndexOutOfBoundsException: begin 4, end 3, length 3
        at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4601)
        at java.base/java.lang.String.substring(String.java:2704)
        at java.base/java.lang.String.substring(String.java:2677)
        at TRANSFORMER/sdlink@1.18.x-version/me.hypherionmc.sdlink.events.ServerEvents.commandEvent(ServerEvents.java:131)
        at net.minecraftforge.eventbus.ASMEventHandler_1013_ServerEvents_commandEvent_CommandEvent.invoke(.dynamic)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.commands.Commands.m_82117_(Commands.java:224)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.server.dedicated.DedicatedServer.m_139665_(DedicatedServer.java:343)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:329)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:808)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:668)
        at TRANSFORMER/minecraft@1.18.1/net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:258)
        at java.base/java.lang.Thread.run(Thread.java:833)

``` 

that occurs when trying to say something from console, but then it does send this 
![image](https://user-images.githubusercontent.com/35115304/147691886-2534e16a-cb13-42a4-bdfd-7c7bd02904ab.png)

I am not receiving messages from discord in Minecraft. 

Also, the Name should not be "Minecraft Server" I changed the web hook name. It should either take the Bot name from discord API, or the webhook name. nor is it taking the Bots or Webhooks Avatar.
Also, you should probably not show 'executed command:'.. 

most bots show this when server says something 
![image](https://user-images.githubusercontent.com/35115304/147692011-0ea356fb-3a9f-409c-8118-07881f1e06c5.png)

**Comment by hypherionmc** [comment]
Looks like a conflict with or something that Architectury is changing. I'll have to look at that code again.

As for the bot name, I actually forgot to add the feature for changing the name to the new version. I'll add it in the next release along with a fix for the first issue

**Comment by hypherionmc** [comment]
As for the "not receiving messages", make sure that the bot has perms to read the channel message history, and also ensure that you have the `channelID` set to the id the channel you are using for chat relay

**Comment by itsdinkd** [comment]
> As for the "not receiving messages", make sure that the bot has perms to read the channel message history, and also ensure that you have the `channelID` set to the id the channel you are using for chat relay

It definitely is the correct channel in the config, as well as the web hook is also on said channel.

**Comment by hypherionmc** [comment]
Reminder to myself: SORT THIS OUT

**Comment by hypherionmc** [comment]
Long overdue.... Sorry about that, but things have been hectic. Both of these issues should be solved with Version 1.2
